cliquet/authorization.py

```diff
@@ -39,6 +39,11 @@ class RouteFactory(object):
 
     def __init__(self, request):
         service = utils.current_service(request)
+
+        # Handle blank requests
+        if service is None:
+            return
```

Comment:
We can probably create an app with a configurator like this to cover this if with tests.